packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -1,17 +1,20 @@
 module.exports = {
   'rules': {
     // enforce spacing inside array brackets
-    'array-bracket-spacing': 0,
+    'array-bracket-spacing': [2, 'never'],
+    // enforce spacing inside single-line blocks
+    // http://eslint.org/docs/rules/block-spacing
+    'block-spacing': [2, 'always'],
```

Comment:
This is going to require re-linting in places that were formerly passing lint, but I like it because it matches what we do in ruby.